### PR TITLE
Update subscription_services_privacy_notice.md

### PR DESCRIPTION
**Pull request by jejohnsonmoz**
Updating for Firefox Accounts > Mozilla Accounts change and other formatting changes. 

This Notice will be in place from 1 Nov 23 to 29 Nov 23, when it will be replaced with a version which also encompasses Monitor

### CODE REVIEW

File: en/subscription_services_privacy_notice.md [hunk 1]
```diff
@@ -1,56 +1,65 @@
 # Mozilla Subscription Services | Privacy Notice
 
-Version 1.0, Effective October 11, 2022
-{: datetime="2022-10-11" }
+Version 1.1, Effective November 1, 2023
+{: datetime="2023-11-1" }
```
Comment (peiying2) [inline en/subscription_services_privacy_notice.md:4]: `{: datetime="2023-11-1" }` should be `{: datetime="2023-11-01" }`

File: en/subscription_services_privacy_notice.md [hunk 2]
```diff
@@ -1,56 +1,65 @@
...
 
-__Masks and where you use them.__ Mozilla maintains a copy of your account information to provide the service, in particular to associate your primary email address with your masked email address(es). If you create a custom mask, Mozilla stores it in order to forward emails to it. Mozilla stores the site where you created the mask, sites where you subsequently use the mask, and any labels associated with the mask in order to ensure that your masks are easily findable when you’re ready to use them. Learn how to [enable and disable these features](https://relay.firefox.com/faq).
+__Firefox Relay Masks (and where you use them).__ Mozilla maintains a copy of your account information to provide the service, in particular to associate your primary email address with your masked email address(es). If you create a custom mask, Mozilla stores it in order to forward emails to it. Mozilla stores the site where you created the mask, sites where you subsequently use the mask, and any labels associated with the mask in order to ensure that your masks are easily findable when you’re ready to use them. Learn how to [enable and disable these features](https://relay.firefox.com/faq).
```
Comment (peiying2) [inline en/subscription_services_privacy_notice.md:46]: lower case `m` in `Masks`?

File: en/subscription_services_privacy_notice.md [hunk 3]
```diff
@@ -1,56 +1,65 @@
...
 
-__Phone calls and text messages.__ To send and forward your phone calls and text messages, Firefox Relay processes your phone number and text messages. We store a log of the phone numbers you’ve contacted through Relay in order to show your call and text logs, send text replies and block phone numbers. We do not monitor or store the content of phone calls you make through Firefox Relay.
+__Firefox Relay Phone calls and text messages.__ To send and forward your phone calls and text messages, Firefox Relay processes your phone number and text messages. We store a log of the phone numbers you’ve contacted through Relay in order to show your call and text logs, send text replies and block phone numbers. We do not monitor or store the content of phone calls you make through Firefox Relay.
```
Comment (peiying2) [inline en/subscription_services_privacy_notice.md:48]: lower case `p` in `Phone`?

File: en/subscription_services_privacy_notice.md [hunk 4]
```diff
@@ -1,56 +1,65 @@
...
 
 Read the telemetry documentation for [Firefox Relay](https://github.com/mozilla/fx-private-relay/blob/main/METRICS.md). You can opt-out of telemetry collection by turning on the [Do Not Track (DNT)](https://support.mozilla.org/kb/how-do-i-turn-do-not-track-feature) feature in your browser.
 
 __Information We Share.__ Firefox Relay shares information with third parties to provide the service to you. Mozilla has contracted with this company requiring them to protect your information. Here’s who we use to support Firefox Relay:
```
Comment (peiying2) [inline en/subscription_services_privacy_notice.md:52]: `Information we share` so the style is consistent with the other bullet points.

**Review by peiying2 — CHANGES_REQUESTED** [review]

File: en/subscription_services_privacy_notice.md [hunk 5]
```diff
@@ -1,56 +1,65 @@
...
 
-* __[Twilio.](https://www.twilio.com)__ Twilio receives your phone number, your phone mask, and the phone numbers that you exchange phone calls and text messages with. Twilio also receives the content of text messages you send and receive through Firefox Relay and Mozilla has set the Twilio service to delete its records of the text messages you send and receive through Firefox Relay after 7 days.
+* __[Twilio.](https://www.twilio.com/en-us/legal/privacy)__ Twilio receives your phone number, your phone mask, and the phone numbers that you exchange phone calls and text messages with. Twilio also receives the content of text messages you send and receive through Firefox Relay and Mozilla has set the Twilio service to delete its records of the text messages you send and receive through Firefox Relay after 7 days.
```
[the remark this answers is not in the document]
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:56]: It looks like it's still this way? I believe the version I'm committing has the en-US for Twilio's page

File: en/subscription_services_privacy_notice.md [hunk 6]
```diff
@@ -1,56 +1,65 @@
...
+Version 1.1, Effective November 1, 2023
+{: datetime="2023-11-1" }
 
 ## At Mozilla, we design products with your privacy in mind.
 
 __Mozilla VPN__ protects your device’s internet connections. Mozilla partners with Mullvad to privately and securely encrypt your internet traffic.
 
-__Firefox Relay__ allows you to keep your primary email address and phone number safe and private from online services by making a unique, random (mask) phone number and email address(es).
+__Firefox Relay__ protects your identity with secure email and phone masking. Our secure and easy to use email and phone masks help keep your identity private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and get only the emails you want in your inbox.  
+
```
[the remark this answers is not in the document]
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:11]: Done

File: en/subscription_services_privacy_notice.md [hunk 7]
```diff
@@ -1,56 +1,65 @@
...
+__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you're using the Mozilla VPN service.  Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
 
-__Network data.__ Mullvad receives your internet traffic to provide the service. When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. Mullvad has committed not to log any of the data it receives. Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
```
[the remark this answers is not in the document]
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:33]: done

File: en/subscription_services_privacy_notice.md [hunk 8]
```diff
@@ -1,56 +1,65 @@
...
+In addition, to help understand the effectiveness of our marketing campaigns, Mozilla may receive your IP address in addition to campaign data and referral data like what ad campaigns you engaged with and your operating system, device type, device identifiers, and operating system version. Mozilla shares this with Adjust, but we do not share or store your IP address. Read the [Adjust documentation](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/adjust/adjust.md).
 
-### Mozilla VPN {: #vpn }
+__Technical data.__ Across Subscription Services, your IP address is temporarily collected as part of our server logs for 90 days. For VPN, Mozilla also receives information about the installed version of your Mozilla VPN client software, and the devices they are installed on, including their operating systems and hardware configuration. We use Technical data to improve performance and stability for our users, and to measure the Services’ performance. In addition, our partner Sentry.io will receive technical data for error and performance monitoring purposes. Learn more at [Sentry.io’s Privacy Policy](https://sentry.io/privacy/).
```
[the remark this answers is not in the document]
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:27]: done

File: en/subscription_services_privacy_notice.md [hunk 9]
```diff
@@ -1,56 +1,65 @@
...
+
+__Location data.__ Mozilla Subscription Services receive your IP address when you sign up for and use the service(s). We use the IP address in order to approximate your location because Mozilla Subscription Services availability, pricing and offers may depend on your country. We also use your IP address for fraud and abuse prevention purposes across our products.
+
+__Firefox Relay email messages.__ To send and forward your email messages from your masked email address(es) to your primary email address, Firefox Relay processes your email messages. We do not read or store the content of any of your messages. In the event that an email cannot be delivered to you, we will keep it on our servers and delete it after it has been delivered (in no event will we hold onto it for more than three days). If you use the feature to block promotional emails, the Service will check email headers to determine whether the email should be blocked.
```
[the remark this answers is not in the document]
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:44]: fixed.

File: en/subscription_services_privacy_notice.md (hunk 1 above)
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:4]: fixed.

File: en/subscription_services_privacy_notice.md (hunk 2 above)
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:46]: I have made the headings' capitalization consistent. I changed m to lowercase and also the p in phone to lowercase

File: en/subscription_services_privacy_notice.md (hunk 3 above)
Comment (mafeld) [reply en/subscription_services_privacy_notice.md:48]: done

**Review by peiying2 — APPROVED** [review]
Thank you for the fixes. We are all set.

**Pull request merged**